test/apiTests.spec.ts

```diff
@@ -0,0 +1,85 @@
+import { test, expect } from '@playwright/test'
+
+const baseURL = `https://simple-books-api.glitch.me`
+
+test.describe('Book api test', () => {
+  let token: string
+  let orderId: string
+
+  test.beforeAll(async ({ request }) => {
+    const response = await request.post(`${baseURL}/api-clients`, {
+      data: {
+        clientName: 'Frank',
+        clientEmail: `test${Date.now()}@example.com`,
+      },
+    })
+
+    expect(response.status()).toBe(201)
+    const body = await response.json()
+    token = body.accessToken
+    expect(token).toBeTruthy()
+  })
+
+  test('Register API client to get token', async ({ request }) => {
+    expect(token).toBeTruthy()
+  })
+
+  test('Submit a new order', async ({ request }) => {
```

Comment:
A small challenge to add spice to this if you want: How would you run this same test for 3 different customerNames?

Reply:
I'm not sure i have time during this week to go back over this but maybe in the future.